src/components/header/header.css

```diff
@@ -0,0 +1,37 @@
+.headerContainer {
+    display: flex;
+    height: 60px;
+    background-color: white;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 35px;
+    padding-right: 20px;
+    box-shadow: 0px 4px 4px lightgray;
+}
+
```

Comment:
generally the ordering of css by convention is this:
width/height
background styles
margins/paddings
display
all the other styling

^ If you can switch up the order of your CSS styles, it would be great! I love how you grouped them together tho.